http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/sender/RedirectChannelInitializer.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Could you update license header year to 2017?

Reply:
done